src/mount/mastercomm.cc

```diff
@@ -1389,6 +1406,13 @@ void fs_statfs(uint64_t *totalspace,uint64_t *availspace,uint64_t *trashspace,ui
 }
 
 uint8_t fs_access(uint32_t inode,uint32_t uid,uint32_t gid,uint8_t modemask) {
+	#ifdef _WIN32
+	// Code specific to Windows
+	if (masterServerIsDisconnected) {
+		return SAUNAFS_STATUS_OK;
+	}
```

Comment:
I don't know if it is correct to return OK if the master is disconnected. Please verify.

Reply:
I checked the behavior and it is the same without that line so I decided to delete it